src/esp/physics/bullet/BulletArticulatedObject.cpp

```diff
@@ -491,6 +491,25 @@ std::vector<float> BulletArticulatedObject::getJointPositions() {
   return positions;
 }
 
+std::vector<float> BulletArticulatedObject::getJointMotorTorques(
+    double fixedTimeStep) {
+  std::vector<float> torques(btMultiBody_->getNumDofs());
+  for (auto& motor : jointMotors_) {
+    auto& settings = motor.second->settings;
+    if (settings.motorType == JointMotorType::SingleDof) {
+      auto& btMotor = articulatedJointMotors.at(motor.first);
+      btScalar impulse = btMotor->getAppliedImpulse(0);
+      float force = (float)impulse / fixedTimeStep;
```

Comment:
Should fix the clang-tidy, tested on CI.
```suggestion
      float force = impulse / float(fixedTimeStep);
```